markdown/usage/installation.md

```diff
@@ -31,125 +18,52 @@ java -version
-unzip master.zip -d ~/my-pipelines/nf-core/
-cd ~/my_data/
-nextflow run ~/my-pipelines/nf-core/YOUR_PIPELINE-master
-```
 
-To stop nextflow from looking for updates online, you can tell it to run in offline mode by specifying the following environment variable in your ~/.bashrc file:
+## Pipeline software
 
-```bash
-export NXF_OFFLINE='TRUE'
-```
-
-### Development
-
-If you would like to make changes to the pipeline, it's best to make a fork on GitHub and then clone the files. Once cloned you can run the pipeline directly as above.
-
-
-## Pipeline configuration
-By default, the pipeline loads a basic resource configuration:
-
-```nextflow
-process {
-
-  cpus = { check_max( 2, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
-
-  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'terminate' }
-  maxRetries = 1
-  maxErrors = '-1'
-
-  // Process-specific resource requirements
-  withName: fastqc {
-    time = { check_max( 8.h * task.attempt, 'time' ) }
-  }
-  withName:toolX {
-    cpus = { check_max( 8, 'cpus' ) }
-    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  }
-  withName: toolY {
-    cpus = { check_max( 10, 'cpus' ) }
-    memory = { check_max( 200.GB * task.attempt, 'memory' ) }
-    time = { check_max( 5.h * task.attempt, 'time' ) }
-  }
-}
-
-params {
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-}
-```
+An analysis pipeline chains the execution of multiple tools together.
+Historically, all tools would have to be manually installed, often a source of great frustration and a key step where reproducibility between analyses is lost.
+All nf-core pipelines fully utilise the built-in support for software packaging that Nextflow offers - namely Docker, Singularity and Conda:
 
-This uses a number of sensible defaults for process requirements and is suitable for running
-on a simple (if powerful!) local server.
+* [Docker](https://docs.docker.com/install/)
+  * Typically used locally / on single-user servers and the cloud.
+  * Analysis runs in a _container_, which behaves like an isolated operating system
+  * Previously required system root access, though a "rootless mode" is available since late 2019
+  * Best option for ultimate reproducibility
+* [Singularity](https://www.sylabs.io/)
+  * Often used as an alternative to Docker on multi-user systems such as HPC systems.
+  * Also runs _containers_ and can create these from Docker images
+  * Does not need root access or any daemon processes - images built from files
+* [Conda](https://conda.io/)
+  * Packaging system that manages environments instead of running analysis in containers.
+  * The software still runs in your operating system environment, theoretically the reproducibility of results is slightly lower than Docker / Singularity due to this.
```

Comment:
Also because the low-level dependencies required to create the environment can change over time.